lib/swaps/Swaps.ts

```diff
@@ -589,7 +592,8 @@ class Swaps extends EventEmitter {
         deal,
         peer,
         reqId,
-        failureReason: SwapFailureReason.UnexpectedClientError,
+        failureReason: (err.message?.includes('insufficient local balance')) ?
```

Comment:
Same thing here, `getRoute` should throw an error indicating insufficient balance, and then we check for that error here when setting the failure reason.